src/lib/containers/widgets/facet-nav/FacetNav.tsx

```diff
@@ -85,7 +85,7 @@ const FacetNav: React.FunctionComponent<FacetNavProps> = ({
       )
       setIsFirstTime(false)
     }
-  }, [data, isFirstTime])
+  }, [data, isFirstTime, facetsToPlot])
```

Comment:
fixed a warning (reference to facetsToPlot in the useEffect without declaring it here)